### PR TITLE
Changecss overhaul example

### PR DESCRIPTION
**Pull request by deanwhillier**
#### Summary
This PR replaces the `changeCSS()` function to using CSS variables for applying theme colours. This PR specifically deals with the `theme.sidebarBg` colour.

There isn't a specific ticket for this PR yet as we are using it as an example of replacing `changeCSS()` with CSS variables.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-23811

### CODE REVIEW

File: sass/components/_scrollbar.scss [hunk 1]
```diff
@@ -15,16 +15,6 @@
...
-    &.app__body {
-        scrollbar-3dlight-color: #7D7E94;
-        scrollbar-arrow-color: #C1C1D1;
-        scrollbar-darkshadow-color: #2D2C4D;
-        scrollbar-face-color: rgba(0, 0, 0, .1);
-        scrollbar-highlight-color: #7D7E94;
-        scrollbar-shadow-color: #2D2C4D;
-        scrollbar-track-color: rgba(0, 0, 0, .1);
-    }
-
```
Comment (deanwhillier) [inline sass/components/_scrollbar.scss:27]: These are IE specific styles, so removing since we don't support IE anymore.

File: utils/utils.jsx [hunk 2]
```diff
@@ -630,8 +623,6 @@ export function applyTheme(theme) {
         changeCss('.app__body .popover.top>.arrow:after, .app__body .tip-overlay.tip-overlay--chat .arrow', 'border-top-color:' + theme.centerChannelBg);
         changeCss('@media(min-width: 768px){.app__body .form-control', 'background:' + theme.centerChannelBg);
         changeCss('.app__body .attachment__content, .app__body .attachment-actions button', 'background:' + theme.centerChannelBg);
-        changeCss('body.app__body', 'scrollbar-face-color:' + theme.centerChannelBg);
-        changeCss('body.app__body', 'scrollbar-track-color:' + theme.centerChannelBg);
```
Comment (deanwhillier) [inline utils/utils.jsx:634]: These are IE specific styles, so removing since we don't support IE anymore.

File: utils/utils.jsx [hunk 3]
```diff
@@ -735,7 +726,6 @@ export function applyTheme(theme) {
         if (!UserAgent.isFirefox() && !UserAgent.isInternetExplorer() && !UserAgent.isEdge()) {
             changeCss('body.app__body ::-webkit-scrollbar-thumb', 'background:' + changeOpacity(theme.centerChannelColor, 0.4));
         }
-        changeCss('body', 'scrollbar-arrow-color:' + theme.centerChannelColor);
```
Comment (deanwhillier) [inline utils/utils.jsx:738]: These are IE specific styles, so removing since we don't support IE anymore.

**Comment by deanwhillier** [comment]
I dealt with each selector in it's own commit if you need to get a better idea of which css addition/changes are part of which removals of `changeCSS()`

**Review by hmhealey — APPROVED** [review]

File: sass/responsive/_mobile.scss [hunk 4]
```diff
@@ -410,7 +410,17 @@
             overflow: auto;
         }
     }
-
+    .app__body {
+        .modal {
+            .settings-modal {
+                &:not(.settings-modal--tabless):not(.display--content) {
```
Comment (nevyangelova) [inline sass/responsive/_mobile.scss:416]: Should these modals depend on '.settings-modal'? Seems like they don't need the styles provided by that class.

**Pull request closed without merging**